src/app/app.component.ts

```diff
@@ -1,16 +1,37 @@
-import { Component } from "@angular/core";
+import { Component, OnInit, OnDestroy } from "@angular/core";
 import { TranslateService } from "@ngx-translate/core";
 import { CartStoreService } from "src/app/@core/stores/cart/cart.store.service";
+import { NgcCookieConsentService, NgcInitializeEvent, NgcStatusChangeEvent, NgcNoCookieLawEvent } from "ngx-cookieconsent";
+import { Subscription } from "rxjs";
 
 @Component({
     selector: "app-root",
     templateUrl: "./app.component.html",
 })
-export class AppComponent {
-    title = "Pard";
-    constructor(private translate: TranslateService, private cartStoreService: CartStoreService) {
+export class AppComponent implements OnInit {
+    constructor(private translate: TranslateService, private cartStoreService: CartStoreService, private ccService: NgcCookieConsentService) {
         this.translate.setDefaultLang("en");
         this.translate.use(this.translate.getBrowserLang());
         this.cartStoreService.syncLocalStorageToListings();
     }
+
+    ngOnInit() {
+        // Needed for translations
+        this.translate
+            .get(["cookie.header", "cookie.message", "cookie.dismiss", "cookie.allow", "cookie.deny", "cookie.link", "cookie.policy"])
+            .subscribe((data) => {
+                this.ccService.getConfig().content = this.ccService.getConfig().content || {};
```

Comment:
Let's save the processing power and do object nesting and function calling just once.
```js
const cookieContent = this.ccService.getConfig();
cookieContent.content = this.ccService.getConfig().content || {};
const content = cookieContent.content;
content.header = data["cookie.header"];
content.message = data["cookie.message"];
content.dismiss = data["cookie.dismiss"];
.
.
.
```